cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -69,34 +128,55 @@ void run_mnist(const std::string& model_name,
                       SmartRedis::MemoryLayout::nested);
     double put_tensor_end = MPI_Wtime();
     delta_t = put_tensor_end - put_tensor_start;
-    timing_file << rank << "," << "put_tensor" << ","
-                << delta_t << std::endl << std::flush;
+    put_tensor_times.push_back(delta_t);
+    //timing_file << rank << "," << "put_tensor" << ","
```

Comment:
I like that we have moved the timings outside of the loop.  Maybe we can remove these old comments?